app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -304,38 +317,29 @@ function AcuantCapture(
 
   /**
    * Calls onChange with next value and resets any errors which may be present.
-   *
-   * @param {Blob|string|null} nextValue Next value.
-   * @param {ImageAnalyticsPayload=} metadata Capture metadata.
    */
-  function onChangeAndResetError(nextValue, metadata) {
+  function onChangeAndResetError(
+    nextValue: Blob | string | null,
+    metadata?: ImageAnalyticsPayload | undefined,
+  ) {
     setOwnErrorMessage(null);
     onChange(nextValue, metadata);
   }
 
   /**
    * Returns an analytics payload, decorated with common values.
-   *
-   * @template {ImageAnalyticsPayload|AcuantImageAnalyticsPayload} P
-   *
-   * @param {P} payload
-   *
-   * @return {P}
    */
-  function getAddAttemptAnalyticsPayload(payload) {
+  function getAddAttemptAnalyticsPayload<P>(payload: P): P {
```

Comment:
Since `@template` specified a type, I think we should preserve it here:

```suggestion
  function getAddAttemptAnalyticsPayload<
    P extends ImageAnalyticsPayload | AcuantImageAnalyticsPayload,
  >(payload: P): P {
```